gui/resources.xml

```diff
@@ -1650,6 +1652,7 @@ Vocabulary
           <property name="icon-name">mypaint-tool-inking-symbolic</property>
           <signal name="activate" handler="mode_flip_action_activated_cb"/>
         </object>
+        <accelerator key="y"/>
```

Comment:
Do you have an alternative key for Inking tool then? I was thinking `<accelerator key="i"/> but that would mean I would need to move the MirrorView keys.